Tests/HTTPStatusCodesTests/RegressionTests.swift

```diff
@@ -84,16 +84,16 @@ final class RegressionTests: XCTestCase {
         XCTAssertEqual(HTTPStatusCode.preconditionRequired, 428)
         XCTAssertEqual(HTTPStatusCode.tooManyRequests, 429)
         XCTAssertEqual(HTTPStatusCode.requestHeaderFieldsTooLarge, 431)
-        XCTAssertEqual(HTTPStatusCode.loginTimeout, 440)
-        XCTAssertEqual(HTTPStatusCode.noResponse, 444)
-        XCTAssertEqual(HTTPStatusCode.retryWith, 449)
+        XCTAssertEqual(HTTPStatusCode.iisLoginTimeout, 440)
+        XCTAssertEqual(HTTPStatusCode.nginxNoResponse, 444)
+        XCTAssertEqual(HTTPStatusCode.iisRetryWith, 449)
         XCTAssertEqual(HTTPStatusCode.unavailableForLegalReasons, 451)
 //        XCTAssertEqual(HTTPStatusCode.requestHeaderTooLarge, 494)
-        XCTAssertEqual(HTTPStatusCode.certError, 495)
-        XCTAssertEqual(HTTPStatusCode.noCert, 496)
-        XCTAssertEqual(HTTPStatusCode.httpToHTTPS, 497)
+        XCTAssertEqual(HTTPStatusCode.nginxSSLCertificateError, 495)
```

Comment:
Why have the names of these changed?

Reply:
Because Xcode reported it as deprecated and renamed.

    'certError' is deprecated: renamed to 'nginxSSLCertificateError'

If I am seeing it correctly, you added the deprecation annotation to `HTTPStatusCode.certError` on 2nd September 2016.

Reply:
I guess my comment should have been, why have these been changed. They've been deprecated but they only produce warnings - they should be changed in the regression test code.

Reply:
Ah, so then I should undo these renames in `RegressionTests`?

Reply:
Yes please! There is a complier flag to suppress the deprecation warnings (`-Wno-deprecated-declarations`) but it only works for the Objective-C regression test - I've never managed to find the corresponding flag for Swift...